### ISSUE TITLE
Necrotizing enterocolitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6180]
**Preferred term label:**
Necrotizing enterocolitis 

**Synonyms**
NEC

**Definition (free text, please give PubMed ID)**
Inflammation of the intestine leading to bacterial invasion causing cellular damage and death which causes necrosis of the colon and intestine. It almost exclusively affects neonates.
PMID: 30020729 

**Parent term (use hpo.jax.org/app)**
Enterocolitis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 605026

**Comment by pnrobinson** [comment]
added as HP_0033165
